dev/com.ibm.ws.jpa.container.v31/bnd.bnd

```diff
@@ -29,16 +24,13 @@ Private-Package: \
   com.ibm.ws.jpa.container.v31.JPAPCtxtAttributeAccessorV31, \
   com.ibm.ws.jpa.container.v31.internal.JPA31Runtime
 
-javac.source: 11
-javac.target: 11
-
 -buildpath: \
-	com.ibm.ws.jpa.container;version=latest,\
-	com.ibm.websphere.javaee.persistence.2.2;version=latest,\
+	com.ibm.ws.jpa.container.jakarta;version=latest,\
+	io.openliberty.jakarta.persistence.3.1;version=latest,\
 	com.ibm.wsspi.org.osgi.core;version=latest,\
 	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
 	com.ibm.tx.core;version=latest,\
```

Comment:
If you still need com.ibm.tx.core, you probably should use com.ibm.tx.jta.jakarta which is the actual transformed bundle since com.ibm.tx.core is consumed by com.ibm.tx.jta.